src/main/java/refactoringml/App.java

```diff
@@ -210,9 +213,8 @@ private Project initProject(Git git) throws GitAPIException, IOException {
 	private void processCommit(RevCommit currentCommit, GitHistoryRefactoringMiner miner, RefactoringHandler handler, RefactoringAnalyzer refactoringAnalyzer, ProcessMetricsCollector processMetrics){
 		long startCommitTime = System.currentTimeMillis();
 		String commitHash = currentCommit.getId().getName();
+		Transaction t = db.beginTransaction();
```

Comment:
What happens if we cannot create a new transaction?

Reply:
I have added a try catch in the method above so it doesn't stop the whole project when one commit fails.